backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -278,6 +297,8 @@ public Parcel PendingUpdate(Parcel parcel)
             var ownsABuilding = originalParcel.Buildings.Any(pb => userAgencies.Contains(pb.Building.AgencyId.Value));
             if (!allowEdit && !ownsABuilding) throw new NotAuthorizedException("User may not edit parcels outside of their agency.");
 
+            if (!isAdmin && originalParcel?.ProjectNumbers?.Contains("SPP") == true) throw new NotAuthorizedException("User may not edit parcels that are in a SPP Project.");
```

Comment:
Don't change, but I suspect this will cause issues down the road.  The `SPP` value is a configurable setting for projects.  In the future there could be all different types of project numbers.